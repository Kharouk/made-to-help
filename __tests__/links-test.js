import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import LinksScreen from '../screens/LinksScreen';

describe('Links snapshot', () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it('renders directions form', async () => {
    const tree = renderer.create(<LinksScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
