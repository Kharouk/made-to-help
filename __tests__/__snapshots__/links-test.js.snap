// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links snapshot renders directions form 1`] = `null`;
